apps/vr-tests/.storybook/main.js

```diff
@@ -14,21 +13,6 @@ module.exports = {
     reactDocgen: false,
   },
   webpackFinal: config => {
-    const tsPaths = new TsconfigPathsPlugin({
```

Comment:
tsconfig paths were only for v9 since the `custom` webpack config used resolve aliases